components/automate-chef-io/content/docs/desktop.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "Desktop Dashboard"
+description = "Using the Desktop Dashboard"
+date = 2018-03-26T16:01:47-07:00
+draft = false
+bref = ""
+toc = true
++++
+
+The Chef Automate _Desktop_ dashboard displays status information about all desktops connected to Chef Automate.
+Desktop information populates this dashboard after a Chef Infra Client run has executed.
+
+## Setting Up the Desktop Dashboard
+
+Enable the Desktop dashboard with: `chef-automate deploy --product automate --product infra-server --product desktop`.
+For more information, see [Install Chef Infra Server with Automate](https://automate.chef.io/docs/infra-server/).
+The Desktop dashboard has no supported compliance profiles, and installation with the `--product desktop` flag includes no compliance profiles.
+
+{{< info >}}
+When installing Chef Automate with the `--product desktop` flag, _Data Lifecycle_ settings will set by default to not mark nodes as missing and to not delete missing nodes.
+We encourage users to not change these specific settings, and not defeat the monitoring purpose of the Desktop dashboard.
+{{< /info >}}
+
+## Desktop Dashboard Display
+
+Within Chef Automate, the _Desktop_ dashboard uses four panels to summarize information: _Daily Check-in_ _Check-in History_, _Top 10 Errors_, and _Time since Last Check-in_.
+Selecting rows within these displays will list relevant desktops and selecting an singular desktop reveals its detailed information.
+
+The _Desktop_ dashboard does not support project filtering.
+Node counts in the _Desktop_ dashboard may include liveness agents.
+
+### Daily Check-in
+
+The _Daily Check-in_ display shows a top-level view of daily desktop check-in statistics.  
+A bar graphic illustrates the proportion of desktops with `unknown` and `checked-in` statuses.
+Below the bar, boxes displays counts for desktops with `unknown` and `checked-in` statuses, and a total count of desktops.
+`Checked-in` refers to desktops reporting into Chef Automate.
+`Unknown` desktops did not report to Chef Automate during the last twenty-four hours.
+
+### Check-in History
+
+_Check-in History_ shows the history of checked-in desktop counts as graphed over a selected period of time.
+Adjust the timescale with the timescale drop-down menu and change the graph display.
+The timescale options are "Last 3 Days," "Last 7 Days," and "Last 14 Days".
+
+### Top 10 Errors
```

Comment:
Reorder these two sections since "Top 10 Errors" appears after "Time Since Last Check-in" in the UI.